packages/sanity/src/core/releases/tool/detail/__tests__/ReleaseDetailsEditor.test.tsx

```diff
@@ -67,7 +67,7 @@ describe('ReleaseDetailsEditor', () => {
       () => {
         expect(useReleaseOperations().updateRelease).toHaveBeenCalledWith(release)
       },
-      {timeout: 250},
+      {timeout: 10_000},
```

Comment:
had to increase timeout here to get the tests to pass again, likely due to coverage instrumentation making things run slower.